PurchasesTests/SubscriberAttributes/SubscriberAttributesManagerTests.swift

```diff
@@ -1333,9 +1304,10 @@ class SubscriberAttributesManagerTests: XCTestCase {
         self.mockAttributionDataMigrator.stubbedConvertAttributionDataToSubscriberAttributesResult = [expectedConversionKey: expectedConvertedValue]
         let expectedAttributionData = ["convert": "any", "to": "something"]
 
-        self.subscriberAttributesManager.convertAttributionDataAndSet(asSubscriberAttributes: expectedAttributionData,
-                                                                      network: .adjust,
-                                                                      appUserID: "user_id")
+        self.subscriberAttributesManager.convertAttributionDataAndSetAsSubscriberAttributes(
```

Comment:
I'm not thrilled about these names, but have no idea what I'd change them to. Maybe this func needs to be broken into two smaller ones that are responsible for just one aspect of this operation (not a suggestion, just thinking). There's nothing you need to do here ☮️

Reply:
hmmm how about setSubscriberAttributesFromAttributionData? or just setSubscriberAttributes and let the parameter label do the explaining?

Reply:
agreed on the name being hard to read. 

I'd always try to keep operations as atomic as possible, so there's no sense for clients of this class to convert attribution data without setting it as subs attributes, it's kind of a bummer to have two calls to the class IMO. 

> how about setAttributesFromAttributionData? or just setAttributes and let the parameter label do the explaining?

I love that, I think it could look pretty neat as
`setAttributes(fromAttributionData attributionData:...`